vignettes/lab_grading.Rmd

```diff
@@ -104,6 +106,21 @@ atoxgr_criteria_ctcv4 %>%
   )
 ```
 <br>
+
+<br>
+Likewise, the list of terms defined in the `{admiral}` metadata to implement NCI-CTCAEv5 is below:
+(Terms identical to NCI-CTCAEv4, except `Hyperglycemia`, `Hyperglycemia (Fasting)` and `Hypophosphatemia`)
+which are not present in NCI-CTCAEv5.
+<br>
```

Comment:
Should we make a note of this in the function as well??

Reply:
What function? `derive_var_atoxgr_dir` ? I don't think so, I would expect users to look in the Vignette to find details of whats implemented.